src/util/color/colorpalette.cpp

```diff
@@ -0,0 +1,55 @@
+#include "colorpalette.h"
+
+namespace {
+
+constexpr mixxx::RgbColor kColorMixxxRed(0xC50A08);
```

Comment:
would it be better to define these in a .h so the default colors are accessible elsewhere?

Reply:
They are moved to a dedicated `predefinedcolorpalettes.h` in #2530.

Reply:
cool